src/test/java/lambda/part1/exercise/Exercise1.java

```diff
@@ -1,84 +1,121 @@
 
 import java.util.Arrays;
+import java.util.Comparator;
 import java.util.List;
 
 import static org.junit.Assert.assertArrayEquals;
 import static org.junit.Assert.assertEquals;
 
 public class Exercise1 {
-
-    @Test
-    public void sortPersonsByAgeUsingArraysSortComparator() {
-        Person[] persons = getPersons();
-
-        // TODO использовать Arrays.sort
-
-        assertArrayEquals(new Person[]{
-            new Person("Иван", "Мельников", 20),
-            new Person("Николай", "Зимов", 30),
-            new Person("Алексей", "Доренко", 40),
-            new Person("Артем", "Зимов", 45)
-        }, persons);
-    }
-
-    @Test
-    public void sortPersonsByAgeUsingArraysSortAnonymousComparator() {
-        Person[] persons = getPersons();
-
-        // TODO использовать Arrays.sort
-
-        assertArrayEquals(new Person[]{
-            new Person("Иван", "Мельников", 20),
-            new Person("Николай", "Зимов", 30),
-            new Person("Алексей", "Доренко", 40),
-            new Person("Артем", "Зимов", 45)
-        }, persons);
-    }
-
-    @Test
-    public void sortPersonsByLastNameThenFirstNameUsingArraysSortAnonymousComparator() {
-        Person[] persons = getPersons();
-
-        // TODO использовать Arrays.sort
-
-        assertArrayEquals(new Person[]{
-            new Person("Алексей", "Доренко", 40),
-            new Person("Артем", "Зимов", 45),
-            new Person("Николай", "Зимов", 30),
-            new Person("Иван", "Мельников", 20)
-        }, persons);
-    }
-
-    @Test
-    public void findFirstWithAge30UsingGuavaPredicate() {
-        List<Person> persons = Arrays.asList(getPersons());
-
-        // TODO использовать FluentIterable
-        Person person = null;
-
-        assertEquals(new Person("Николай", "Зимов", 30), person);
-    }
-
-    @Test
-    public void findFirstWithAge30UsingGuavaAnonymousPredicate() {
-        List<Person> persons = Arrays.asList(getPersons());
-
-        // TODO использовать FluentIterable
-        Person person = null;
-
-        assertEquals(new Person("Николай", "Зимов", 30), person);
-    }
-
-    private Person[] getPersons() {
-        return new Person[]{
-            new Person("Иван", "Мельников", 20),
-            new Person("Алексей", "Доренко", 40),
-            new Person("Николай", "Зимов", 30),
-            new Person("Артем", "Зимов", 45)
-        };
-    }
+	
+	@Test
+	public void sortPersonsByAgeUsingArraysSortComparator() {
+		Person[] persons = getPersons();
+		
+		class PersonAgeComarator implements Comparator<Person> {
+			
+			@Override
+			public int compare(Person o1, Person o2) {
+				return Integer.compare(o1.getAge(), o2.getAge());
+			}
+		}
+		// TODO использовать Arrays.sort
+		
+		
+		assertArrayEquals(new Person[]{
+				new Person("Иван", "Мельников", 20),
+				new Person("Николай", "Зимов", 30),
+				new Person("Алексей", "Доренко", 40),
+				new Person("Артем", "Зимов", 45)}, persons);
+		Arrays.parallelSort(persons);
```

Comment:
Why do you sort persons after the assertion?

Reply:
+